.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+# Contributing
+
+
+## Pull Request Merge Requirements
+
+- 2 approvals from other developers (one of them being @hermfischer-wf)
+- QA review
+
+Note: We will be working in the future towards building out a testing infrastructure where tests will be required
```

Comment:
nit: this reads a bit hand wavy to me. Like maybe it'll happen someday. I'd suggest:
`Note: Arelle will be integrating with a CI/CD pipeline and testing infrastructure that once implemented will require passing tests for all PRs.`